sources/packages/backend/apps/db-migrations/src/sql/Queue/Add-assessment-workflow-queue-retry.sql

```diff
@@ -0,0 +1,14 @@
+INSERT INTO
+    sims.queue_configurations(queue_name, queue_configuration)
+VALUES
+    (
+        'assessment-workflow-queue-retry',
+        '{
+            "cron": "0 0 * * * ?",
```

Comment:
why do we need `?` char in the expression?

Reply:
![image](https://github.com/bcgov/SIMS/assets/78114138/d6de3eee-f251-4f59-92f1-ce1eccbd117c)

The current cron expresion is described below:
<pre>
0    0    *    *    *    ?
┬    ┬    ┬    ┬    ┬    ┬
│    │    │    │    │    |
│    │    │    │    │    └ each day of the week
│    │    │    │    └───── every month
│    │    │    └────────── every day of the month
│    │    └─────────────── every hour
│    └──────────────────── 0 minute
└───────────────────────── 0 second
</pre>

The same could be achieved with: "0    0    *    ?    *    *"

More about cron expressions on: https://www.freeformatter.com/cron-expression-generator-quartz.html